next/components/ui/Upload/Upload.tsx

```diff
@@ -42,11 +45,13 @@ export const Upload = ({
         onDragLeave={() => setIsInArea(false)}
       >
         <input
-          id="file_input"
+          id={id}
           ref={fileInputRef}
           type="file"
           className="absolute h-full w-full cursor-pointer opacity-0"
+          aria-invalid={hasError}
           aria-required={required}
+          aria-errormessage={errorMessage ? `${id ?? ''}_err` : ''}
```

Comment:
Please change `''` to `undefined` because empty string is not valid value for `aria-errormessage` according to html validator.

```
aria-errormessage={errorMessage ? `${id ?? ''}_err` : undefined}
```